### ISSUE TITLE
it simply doesn't open

### ISSUE THREAD

**Issue by itsthemaths** [R3nzTheCodeGOD/R3nzSkin#891]
I click on the injector and nothing happens, I've tried everything I know so far and nothing has helped

**Comment by Pyleky** [comment]
something is wrong with your pc. The injector should ask for admin permission before starting, if it doesn't ask for the permission, it has nothing to do with the app.

**Comment by itsthemaths** [comment]
> algo está errado com seu pc. O injetor deve pedir permissão de administrador antes de iniciar, se não pedir permissão, não tem nada a ver com o aplicativo.

he asks and I accept.

**Comment by Adamekg59** [comment]
My application was not opening/ was getting deleted because of antivirus issue. Check if you have your antivirus on.

**Comment by hotline1337** [comment]
Disable your anti-virus, add the folder to the exclusion list, run the injector as administrator.

**Comment by itsthemaths** [comment]
> Desative seu antivírus, adicione a pasta à lista de exclusão, execute o injetor como administrador.

I've already done all this, the problem is that when I click on the launcher it doesn't start, but it keeps all the files there and nothing is deleted
